### PR TITLE
Adding Volvo vehicles #36

### PR DESCRIPTION
**Pull request by droidttk**

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -5722,6 +5722,122 @@ const allSoloCars = {
       'all': ['hs', 'fsp', 'dp'],
     },
   },
+  'Volvo': {
+    'S60 & V60 Polestar': {
```
Comment (Bjorn248) [inline common.js:5726]: These should be separate models

File: common.js [hunk 2]
```diff
@@ -5722,6 +5722,122 @@ const allSoloCars = {
...
+    'C30': {
+      '2006': ['hs', 'dsp'],
+      '2007': ['hs', 'dsp'],
+      '2008': ['hs', 'dsp'],
+      '2009': ['hs', 'dsp']
+    },
+    'NOC': {
+      'all': ['hs']
+    },
+    'S60R': {
```
Comment (Bjorn248) [inline common.js:5751]: Combine this with the previous S60R, valid classes are `gs`, `stu`, `asp`. Remove the other duplicate models.

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Some models should be merged, also remove invalid class.

File: common.js [hunk 3]
```diff
@@ -5722,6 +5722,122 @@ const allSoloCars = {
...
+      '2020': ['ds']
+    },
+    'S60R (except Polestar)': {
+      'all': ['gs']
+    },
+    'V70R (except Polestar)': {
+      'all': ['gs']
+    },
+    'Turbo models (NOC)': {
+      'all': ['gs', 'gt']
```
[the remark this answers is not in the document]
Comment (droidttk) [reply common.js]: Corrected!

File: common.js (hunk 1 above)
Comment (droidttk) [reply common.js]: Corrected!

File: common.js (hunk 2 above)
Comment (droidttk) [reply common.js]: Corrected!

**Pull request merged**